src/web/components/Input/RadioInput.spec.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable testing-library/no-unnecessary-act */
```

Comment:
Don't just disable lint warnings for the whole file - this stops the linter from finding actual problems which might be introduced later.
It looks like the lint rule is wrong this time (something to do with validation in react-hook-form requires it) but the docs all warn that most of the time, it's telling you about something that's actually wrong with your test.

Reply:
You are right, i found the issue is actually using await in front of the userEvent. I've removed the act